README.md

```diff
@@ -37,6 +37,17 @@ Performs extraction of known archive types and e-mail attachments. Produces "raw
 
 First of all, make sure you have setup the core system: https://github.com/CERT-Polska/karton
 
+In order to unpack all available formats you'll also need a few native dependencies that sflock relies on, so make sure you install them with:
+```shell
+RUN sed -i 's/ main/ main non-free/' /etc/apt/sources.list \
```

Comment:
Telling people to run sed on `/etc/apt/sources.list` is a bit shady :thinking:. But I guess technical people will know what it does and adjust accordingly?

Reply:
This command was copied directly from sflocks readme, seems kinda sketchy though. Should we omit it and just make the user worry about switching to non-free?

Reply:
Maybe add `(copied from sflock readme)`. That way, when something breaks it's not technically out fault :duck: